src/components/Application/BriefOwnerHeader.tsx

```diff
@@ -56,6 +68,55 @@ const BriefOwnerHeader = (props: BriefOwnerHeaderProps) => {
         setAnchorEl(null);
     };
 
+    const getBalance = useCallback(async (walletAddress: string, currency_id: number) => {
+        try {
+            const imbueApi = await initImbueAPIInfo();
+            const chainService = new ChainService(imbueApi, user);
+
+            if (!walletAddress) return "No Wallet Found"
```

Comment:
can we get rid of this hard-coded constant string?

Reply:
This might get complicated since it is an async function and async function returns promise but we just want the balance as string

Reply:
resolved